src/components/panel/annotations/AlignAnnotationsList.tsx

```diff
@@ -20,8 +20,62 @@ const AlignAnnotationsList: FC = () => {
 
   const ref = useRef()
 
+
+  function isClickedElAnnotation(event) {
+    // conditions to determine whether clicking inside annotation el (the target has 'data-annotation' attribute as well
+    // therefore I check whether having 'data-cy'= 'annotation'
+    if (event.target.getAttribute('data-cy') === 'annotation') return true
+    const parentEl = event.target.parentElement
+    return parentEl?.getAttribute('data-cy') === 'annotation'
+  }
+
+  async function isClickedElTarget(textContainerEl: HTMLElement, clickedEl: HTMLElement, numberIterations: number) {
+    // TODO: maybe get a useful info from backend of identifying targetEl ?
+    if (numberIterations === 0) return false
+    if (clickedEl.hasAttribute('data-annotation')) return true
+    await isClickedElTarget(textContainerEl, clickedEl.parentElement, numberIterations - 1)
+  }
+
   useEffect(() => {
     if (selectedAnnotation) trackTopChange()
+
+    async function deselectAnnotationOnOutsideClick(event) {
+      const panelEl = document.getElementById(panelId) as HTMLElement
+      let clickedAnnotationId = ''
+
+      // if we click at a target then return - reason: want to deal with target in TextRenderer
+      const textContainerEl = panelEl.querySelector('div[data-text-container="true"]') as HTMLElement
+      if (textContainerEl.contains(event.target) && await isClickedElTarget(textContainerEl, event.target, 5)) return
+
+      if (isClickedElAnnotation(event)) {
+        clickedAnnotationId = event.target.getAttribute('data-annotation') ?? event.target.parentElement?.getAttribute('data-annotation')
+      }
+
+      if (!selectedAnnotation && clickedAnnotationId) {
+        // if event takes place at an annotation and !selectedAnnotation -> we deselected the annotation -> initializeElements
+        initializeElements()
+        return
+      }
+
+      if (!selectedAnnotation) return  // (inferred from above condition) - if no annotation is selected and what we clicked is not an annotation
+      //  -> we do not deselect the 'selectedAnnotation'
+      if (clickedAnnotationId) return   // when clicking at another annotation we do not deselect
+
+      // we preserve the selected annotation in List Mode while switching to Align Mode -> return
+      const annotationsModeToggleEl = panelEl?.querySelector('div[data-cy="annotations-mode-toggle"]') as HTMLElement
```

Comment:
Please don't use cy attributes in business code, they are only meant for testing and will be removed in production builds. In fact, this is one of the flaws of that technique.